packages/components/Chip/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@fluentui-react-native/chip",
+  "version": "0.1.0",
+  "description": "A cross-platform Chip component using the Fluent Design System. A chip is a compact representations of entities (most commonly, people) that can be typed in, deleted or dragged easily.",
+  "license": "MIT",
+  "author": "Microsoft <fluentuinativeowners@microsoft.com>",
+  "homepage": "https://github.com/microsoft/fluentui-react-native",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "depcheck": "fluentui-scripts depcheck",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/adapters": "^0.11.0",
+    "@fluentui-react-native/framework": ">=0.10.0 <1.0.0",
+    "@fluentui-react-native/icon": "^0.18.1",
+    "@fluentui-react-native/interactive-hooks": ">=0.22.30 <1.0.0",
+    "@fluentui-react-native/text": ">=0.20.1 <1.0.0",
+    "@fluentui-react-native/theme-tokens": "^0.25.0",
+    "@fluentui-react-native/tokens": "^0.21.0",
+    "@fluentui-react-native/use-styling": "^0.10.0"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/eslint-config-rules": "^0.1.1",
+    "@fluentui-react-native/scripts": "^0.1.1",
+    "@fluentui-react-native/test-tools": ">=0.1.1 <1.0.0",
+    "@office-iss/react-native-win32": "^0.71.0",
+    "react": "18.2.0",
+    "react-native": "^0.71.0",
+    "react-native-svg": "^13.7.0"
```

Comment:
Before check in, can we double check these are all still accurate so the pipeline doesn't break?

Reply:
Updated, thanks for catching